src/presentation/http/router/note.test.ts

```diff
@@ -50,3 +50,62 @@ describe('Note API', () => {
     });
   });
 });
+
+describe('Note API', () => {
```

Comment:
duplicated test suite

Reply:
What should the name of the test suite ideally be? Will the "Note id API" be suitable?

Reply:
> duplicated test suite

What should the name of the test suite ideally be? Will the "Note id API" be suitable?

Reply:
@elizachi please remove your `describe('Note API', () => {})` wrapper and place all your tests inside existing `describe('Note API', () => {})` wrapper